2022/30xxx/GSD-2022-30190.json

```diff
@@ -48,7 +50,15 @@
             },
             {
                 "type": "WEB",
-                "url": "https://twitter.com/gossithedog/status/1530844804061573120?s=11&t=AtEKru-rQV8pr8LbmPRqxQ"
+                "url": "https://twitter.com/gossithedog/status/1530844804061573120"
+            },
+            {
+                "type": "WEB",
+                "url": "https://app.any.run/tasks/713f05d2-fe78-4b9d-a744-f7c133e3fafb/"
+            },
+            {
+                "type": "WEB",
+                "url": "https://twitter.com/MalwareJake/status/1531022209011048450"
```

Comment:
Single command mitigation (untested for second-order effects)

`reg delete hkcr\ms-msdt /f`